doc/source/whatsnew/v0.17.0.txt

```diff
@@ -100,9 +103,10 @@ Old                     New
 ``convert_timedelta``   ``timedelta``
 =====================   =============
 
-- Coercing types with ``DataFrame.convert_objects`` is now implemented using the
+Coercing types with ``DataFrame.convert_objects`` is now implemented using the
 keyword argument ``coerce=True``.  Previously types were coerced by setting a
-keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
+keyword argument to ``'coerce'`` instead of ``True``, as in
+``convert_dates='coerce'``.
 
   .. ipython:: python
```

Comment:
all of the ``.. ipython:: python`` need to be moved left to the margin (so that there will be no indentation)